internal/policygen/iam.go

```diff
@@ -85,3 +126,26 @@ func validate(instance map[string]interface{}, mandatoryFields []string) error {
 	}
 	return nil
 }
+
+func projectNumber(rn runner.Runner, id string) (string, error) {
+	cmd := exec.Command("gcloud", "projects", "describe", id, "--format", "json")
```

Comment:
You can actually replace this with
```go
cmd := exec.Command("gcloud", "projects", "describe", id, "--format='value(projectNumber)'")
```

Reply:
I actually debated a little between the two but decided to choose json format as I found it's more structured and somewhat feels a bit more "reliable". Another reason is I might need other fields from the `projects describe` output, so the full `json` output can be more robust in that case.

Reply:
I see. I would still argue for just using `value()`; you don't need other fields yet, and it would simplify this function a lot, since it's only returning the `projectNumber`. It can more or less become just 
```
func projectNumber(...) (...) {
    return rn.CmdOutput(exec.Command("gcloud", "projects", "describe", id, "--format='value(projectNumber)'")
}
```

(depending on if you want to check that it's not empty)

Reply:
Changed. Although it's not that simply as there is still post-processing needs to be done.